apps/techlabblog/content/publication/posts/javascript-generators.mdx

```diff
@@ -11,9 +13,9 @@ Generators are a special type of function in JavaScript that can be paused and r
 
 ### Syntax
 
-A generator function is defined using the `function*`syntax, where the asterisk (\*) distinguishes it from a regular function. Inside the function body, the yield keyword is used to pause the function and return a value.
+A generator function is defined using the `function*` syntax, where the asterisk (\*) distinguishes it from a regular function. Inside the function body, the yield keyword is used to pause the function and return a value.
 
-```javascript
+```js showLineNumbers {1-3,4} caption="Caption is here" title="Title is here"
```

Comment:
```suggestion
```js showLineNumbers {1-3,4} caption="Generator Example" title="Syntax"
```

Reply:
Yeah, this was just to show it's possible to show caption/title... This whole blogspot will be deleted anyways.